### PR TITLE
Testing - Packaging v2

### PR DESCRIPTION
**Pull request by Salamandar**
## Problem

- *Description of why you made this PR*

## Solution

- *And how do you fix that problem*

## PR Status

- [ ] Code finished and ready to be reviewed/tested
- [ ] The fix/enhancement were manually tested (if applicable)

## Automatic tests

Automatic tests can be triggered on https://ci-apps-dev.yunohost.org/ *after creating the PR*, by commenting "!testme", "!gogogadgetoci" or "By the power of systemd, I invoke The Great App CI to test this Pull Request!". (N.B. : for this to work you need to be a member of the Yunohost-Apps organization)

### CODE REVIEW

**Comment by Salamandar** [comment]
!testme

File: manifest.toml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+packaging_format = 2
+
+id = "misskey"
+name = "Misskey"
+description.en = "Microblogging platform"
+description.fr = "Platforme de Microblogging"
+
+version = "12.119.2~ynh1"
+
+maintainers = [""]
```
Comment (alexAubin) [inline manifest.toml:12]:
```suggestion
maintainers = []
```

File: tests.toml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+#:schema https://raw.githubusercontent.com/YunoHost/apps/master/schemas/tests.v1.schema.json
+
+test_format = 1.0
+
+[default]
+
```
Comment (alexAubin) [inline tests.toml:6]:
```suggestion
    exclude = ["change_url"]
```

**Comment by alexAubin** [comment]
!testme

**Pull request merged**